### PR TITLE
Update Total Users Tile 

### PR DESCRIPTION
**Pull request by Keitech**
Numbers are relevant to the capacity of the building and total number of users in the building.

### CODE REVIEW

File: src/components/Dashboard/Dashboard.tsx
Comment (dannyshwan) [inline src/components/Dashboard/Dashboard.tsx:94]: cleanup

File: src/components/Dashboard/Tiles/TotalUsers/TotalUsers.tsx
Comment (dannyshwan) [inline src/components/Dashboard/Tiles/TotalUsers/TotalUsers.tsx:12]: Rename this to "CurrentUserCapacityProp" or something

File: src/store/facilityList/reducers.ts
Comment (dannyshwan) [inline src/store/facilityList/reducers.ts:10]: change variable name (initialPerson already exists in personlist reducer)

File: src/store/facilityList/types.ts
Comment (dannyshwan) [inline src/store/facilityList/types.ts:9]: probably would be better to change "size" to currentPopulation, "cap" to capacity. This will help with clarity

File: src/store/facilityList/reducers.ts
Comment (dannyshwan) [inline src/store/facilityList/reducers.ts:10]: "size" -> currentPopulation
"cap" -> capacity

File: src/components/Dashboard/Tiles/TotalUsers/TotalUsers.tsx
Comment (dannyshwan) [inline src/components/Dashboard/Tiles/TotalUsers/TotalUsers.tsx:20]: change "currentCapacity" to "currentPopulation"

File: src/components/Dashboard/Dashboard.tsx
Comment (dannyshwan) [inline src/components/Dashboard/Dashboard.tsx:94]: Cleanup

**Review by dannyshwan — CHANGES_REQUESTED** [review]
Looks good, a few naming changes are needed for clarity

File: src/components/Dashboard/Dashboard.tsx
Comment (dannyshwan) [inline src/components/Dashboard/Dashboard.tsx:94]: cleanup

File: src/components/Dashboard/Tiles/TotalUsers/TotalUsers.tsx
Comment (dannyshwan) [inline src/components/Dashboard/Tiles/TotalUsers/TotalUsers.tsx:12]: Rename this to "CurrentUserCapacityProp" or something

File: src/components/Dashboard/Tiles/TotalUsers/TotalUsers.tsx
Comment (dannyshwan) [inline src/components/Dashboard/Tiles/TotalUsers/TotalUsers.tsx:20]: change "currentCapacity" to "currentPopulation"

File: src/store/facilityList/reducers.ts
Comment (dannyshwan) [inline src/store/facilityList/reducers.ts:10]: "size" -> currentPopulation
"cap" -> capacity

File: src/store/facilityList/types.ts
Comment (dannyshwan) [inline src/store/facilityList/types.ts:9]: - Change currentCapacity to currentPopulation
- probably would be better to change "size" to currentPopulation, "cap" to capacity. This will help with clarity

File: src/store/facilityList/actions.ts
Comment (guptaharsh25) [inline src/store/facilityList/actions.ts:11]: nitpick but do you need `as const` here? I haven't seen this notation before so just clarifying.

File: src/store/facilityList/reducers.ts
Comment (guptaharsh25) [inline src/store/facilityList/reducers.ts:9]: in my opinion these should initially be set to `undefined`. This way, if these vars are `undefined` we know there was an issue with the fetch.

File: src/store/facilityList/reducers.ts
Comment (guptaharsh25) [inline src/store/facilityList/reducers.ts:7]: why is this called `initialPerson`

File: src/store/facilityList/types.ts
Comment (guptaharsh25) [inline src/store/facilityList/types.ts:9]: what is the difference between these two?

Comment (guptaharsh25) [reply src/store/facilityList/types.ts:9]: i don't think i saw a reference to the `facility` interface.

**Review by guptaharsh25 — CHANGES_REQUESTED** [review]
some nitpicks and questions

File: src/components/Dashboard/Tiles/TotalUsers/TotalUsers.tsx
Comment (guptaharsh25) [inline src/components/Dashboard/Tiles/TotalUsers/TotalUsers.tsx:15]: these should be set to `number` no?

File: src/store/facilityList/types.ts
Comment (guptaharsh25) [inline src/store/facilityList/types.ts:4]: should be type `number`

**Review by guptaharsh25 — CHANGES_REQUESTED** [review]
could you post a screenshot showing this works? Just for documentation + testing purposes

File: src/store/facilityList/types.ts
Comment (Keitech) [reply src/store/facilityList/types.ts]: had to also change reducer.ts objects to 0 for the type to be number in type.ts and TotalUsers.tsx

Heres the dashboard with the data from: /facility/by/602ea8d423a00b4812b77ee6

Screen Shot 2021-02-21 at 2 08 32 AM

Comment (Keitech) [reply src/store/facilityList/types.ts]: will also fix styling in another pr

File: src/components/Dashboard/Tiles/TotalUsers/TotalUsers.tsx
Comment (dannyshwan) [inline src/components/Dashboard/Tiles/TotalUsers/TotalUsers.tsx:31]: Change this to "Current Population"

**Review by dannyshwan — APPROVED** [review]
Last little nitpicks from me, otherwise looks good. Wait for Harsh's approval

**Review by guptaharsh25 — APPROVED** [review]

File: src/components/Dashboard/Tiles/TotalUsers/TotalUsers.tsx
Comment (Keitech) [reply src/components/Dashboard/Tiles/TotalUsers/TotalUsers.tsx]: Screen Shot 2021-02-21 at 12 26 46 PM

voila now onto the styling

**Pull request merged**
